README.md

```diff
@@ -1,2 +1,50 @@
-# react-bookstore-shubh
-This app will keep track of books in a library 
+# Book Store
+
+> The website allows users to keep record of books and update book data.
+
+## Built With
+
+- Major languages: JavaScript, HTML, CSS
+- linters
+- Frameworks: React
+
+## Live Demo (if available)
+
+## Screenshots
+
+<hr>
+
+## Getting Started
+**To get the content of this project locally you need to run this command in your terminal:**
+
+[BookStore](https://github.com/Shubh-Dev/react-bookstore-shubh.git)
+
+After cloning In the project directory, you can run:
+
+### `npm start`
+
```

Comment:
- [OPTIONAL ] Please it will be nice to add explicit steps to clone and run your project locally, as this will be of help to non-technical users. Please add descriptive steps from cloning your app to seeing the results displayed on the browser. You can put the steps to clone and run the project in a new `Getting Started` section. This includes `git clone URL` , `cd project_name`, `npm install`, `npm start`, etc
- The README file It is the first file a person will see when they encounter your project, so it should be fairly brief but detailed. It will make your project stand out from a bunch of other